consensus/polybft/blockchain_wrapper.go

```diff
@@ -0,0 +1,191 @@
+package polybft
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/0xPolygon/polygon-edge/blockchain"
+	"github.com/0xPolygon/polygon-edge/consensus"
+	"github.com/0xPolygon/polygon-edge/state"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/umbracle/ethgo"
+	"github.com/umbracle/ethgo/contract"
+)
+
+// // ethereumBackend is an interface that wraps the methods called on ethereum backend protocol
+// type ethereumBackend interface {
+// 	// PeersLen returns the number of peers the node is connected to
+// 	PeersLen() int
+// 	// TxPool returns the current transaction pool
+// 	TxPool() *core.TxPool
+// 	// Broadcast block broadcasts the newly inserted block to the rest of the peers
+// 	BroadcastBlock(block *types.Block, propagate bool)
+// 	// GenesisGasLimit returns the initial gas limit for a block
+// 	GenesisGasLimit() uint64
+// }
```

Comment:
Leftover?

Reply:
Changed in another pr https://github.com/0xPolygon/polygon-edge/pull/789